README.md

```diff
@@ -71,83 +84,42 @@ To build the app from source:
+1. In your clone, open `source/SquadLeader/app/src/main/res/values/strings.xml` and adjust the values of `clientId` and `licenseString` as needed. Refer to [the documentation on licensing an ArcGIS Runtime app](https://developers.arcgis.com/android/guide/license-your-app.htm) for details. Note that the 4.0.0 release of Squad Leader uses only Runtime Basic functionality in ArcGIS Runtime 10.2.6, which means the license string is optional. A valid client ID will disable the "Licensed for Developer Use Only" watermark that otherwise appears on the map.
+1. Follow [Building with Android Studio](#building-with-android-studio) or [Building with Gradle](#building-with-gradle) below.
+
+### Building with Android Studio
+
+1. Start Android Studio and open the Squad Leader project by choosing **Open an existing Android Studio project** on the start screen. In the **Open File or Project** dialog, navigate to your clone, choose `source/SquadLeader`, and click OK.
+    1. Android Studio may present a dialog entitled **Unregistered VCS root detected**. If so, choose **Add root** if you would like Android Studio to work with Git and GitHub.
+1. For convenience, open the project tool window by choosing **View > Tool Windows > Project** if it is not already visible.
+1. In Android Studio, click the **Sync Project with Gradle Files** button, or choose **Tools > Android > Sync Project with Gradle Files**. Wait a few seconds for the Gradle sync to complete.
+1. In Android Studio, click the **Make Project** button, or type Ctrl-F9. Wait a few seconds for the build to complete.
+1. To run the app, connect an Android device by USB, then click the **Run** button, or type Shift-F10.
+1. To generate an Android installer (APK) file, go to **Build > Generate Signed APK**.
+
+### Building with Gradle
+
+1. Open a command prompt and navigate to your clone. Inside your clone, navigate to `source/SquadLeader`.
+1. In the command prompt, set the ANDROID_HOME environment variable to the location of your Android SDK installation. This is likely to be `C:\Users\<username>\AppData\Local\Android\sdk`. (You can set this variable in your system before starting the command prompt if desired.)
+1. On Windows, run `gradlew.bat assemble`. On Linux, run `./gradlew assemble`.
+1. When the build successfully completes, `.apk` files are written to `source/SquadLeader/app/build/outputs/apk`.
+
+## Running Unit Tests
+
+### Running Unit Tests using Android Studio
+
+1. Follow the [Building from Source](#building-from-source) and [Building with Android Studio](#building-with-android-studio) steps above, except that there is no need to run the app or generate an APK file.
+1. In Android Studio, in the Select Debug/Run Configuration dropdown, change from **app** to **All Tests**.
```

Comment:
I did not have this option (just "app") - is there something I needed to do first (besides build app)?